### ISSUE TITLE
Motivation text update

### ISSUE THREAD

**Issue by SnirAttia** [digitalcrust/digitalcrust-website#19]
The introductory and Motivation text on the front page (https://digitalcrust.org/) and the Motivation page (https://digitalcrust.org/motivation) is missing some discussion about the need for ground-up/grassroots development.

I'll suggest minor updates to the text.

**Comment by davenquinn** [comment]
I’ll merge this for now but I think we need to be a little more explicit about what we mean here. I think there needs to be a fairly incisive discussion of the lack of movement from EarthCube and other more top-down attempts at this. So perhaps this is deserving of its own page…

**Comment by SnirAttia** [comment]
Essentially, I agree and think that (1) this could be part of the https://digitalcrust.org/software-as-collaborative-science page but (2) should focus more on why bottom-up is the best/only way to move forward rather than why top-down previous efforts have failed.

Having said that, we do need to have a sense of why things have failed so far:
(1) unsustainable development framework: asking too much of geoscientists who also develop or not having enough development time or long-term development support
(2) funding challenges
(3) breaks in digital data pipeline become nearly 100% leaky bottleneck
(4) lack of incentives/credit for academic researchers who develop things
(5) community does not have the skills and perspective needed to engage with/support efforts
(6) a criminally irresponsible culture around data usability, sharing, and transparency in geology

**Comment by SnirAttia** [comment]
Was thinking about how to address this further and have a quick question @davenquinn.
-I see a link to [[software-as-collaborative-science]] page under content/ which doesn't yet exist, and also a [[Software-driven collaboration]] page which does exist under content/software/. Which do we want to keep? Both? What would be the content differences?
-Similarly, I see the draft page content/__drafts__/Software collaboration which is linked in [[Motivation]] but has essentially the same text. Thoughts?

**Comment by SnirAttia** [comment]
I also feel like #24 is really related to these threads. I thought the report you linked (https://library.ucsd.edu/dc/object/bb5606891b) was quite interesting and the issues of sustainability, grassroots development, governance, and leadership tie in closely to the why of DigitialCrust

**Comment by davenquinn** [comment]
Yeah thanks for all of these detailed thoughts. I agree conceptually that dividing software projects into their types isn't necessarily the most valuable to us. I think it's a sort of limited view, focused around operational funding concerns more than iron conceptual boundaries.

What I was abortively trying to express in the half-written and duplicative pages you discuss above was a sense that, although we want to collaborate better/more widely between subfields, most scientists' points of view are captured by a single way of thinking (or several complementary ways). And this is a feature, not a bug. In complex systems such as the Earth's, we need heuristics to make sense of it. Software allows us to encode some of our assumptions into our tools. And if we can agree between subfields how information should be stored/transferred/used in one instance, that's essentially two worldviews about science coming together. Repeat this a bit, and the software itself begins to encode some of the connections between fields we'd like to build.

Overall, @SnirAttia, I think you have a good sense of what you want to do here, so if you want to make a first pass at some changes, be my guest.
